wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfo.java

```diff
@@ -86,52 +86,96 @@ public void setFirstline(String firstline)
+        if (String.valueOf(this.hasHeader()) != null)
+            sb.append("hasHeader:").append(String.valueOf(this.hasHeader())).append("\n");
+        if (String.valueOf(this.getIsCompressed()) != null)
+            sb.append("IsCompressed:").append(String.valueOf(this.getIsCompressed())).append("\n");
+        if (String.valueOf(this.getIsSuperfile()) != null)
+            sb.append("IsSuperfile:").append(String.valueOf(this.getIsSuperfile())).append("\n");
+        if (this.getJobName() != null)
+            sb.append("JobName:").append(this.getJobName()).append("\n");
+        if (this.getMaxRecordSize() != null)
+            sb.append("MaxRecordSize:").append(this.getMaxRecordSize()).append("\n");
+        if (this.getModified() != null)
+            sb.append("Modified:").append(this.getModified()).append("\n");
+        if (this.getName() != null)
+            sb.append("Name:").append(this.getName()).append("\n");
+        if (this.getNodeGroup() != null)
+            sb.append("NodeGroup:").append(this.getNodeGroup()).append("\n");
+        if (this.getNumParts() != null)
+            sb.append("NumParts:").append(this.getNumParts()).append("\n");
+        if (this.getOwner() != null)
+            sb.append("Owner:").append(this.getOwner()).append("\n");
+        if (this.getPathMask() != null)
+            sb.append("PathMask:").append(this.getPathMask()).append("\n");
+        if (this.getPersistent() != null)
+            sb.append("Persistent:").append(this.getPersistent()).append("\n");
+        if (this.getPrefix() != null)
+            sb.append("Prefix:").append(this.getPrefix()).append("\n");
+        if (this.getRecordCount() != null)
+            sb.append("RecordCount:").append(this.getRecordCount()).append("\n");
+        if (this.getRecordSize() != null)
+            sb.append("RecordSize:").append(this.getRecordSize()).append("\n");
+        if (this.getShowFileContent() != null)
+            sb.append("ShowFileContent:").append(this.getShowFileContent()).append("\n");
+        if (this.getStat() != null)
+            sb.append("Stat:").append(this.getStat()).append("\n");
+        if (this.getSubfiles() != null)
+            sb.append("Subfiles:").append(this.getSubfiles()).append("\n");
+        if (this.getSuperfiles() != null)
+            sb.append("Superfiles:").append(this.getSuperfiles()).append("\n");
+        if (this.getUserPermission() != null)
+            sb.append("UserPermission:").append(this.getUserPermission()).append("\n");
+        if (this.getWuid() != null)
+            sb.append("Wuid:").append(this.getWuid()).append("\n");
+        if (this.getZipFile() != null)
+            sb.append("ZipFile:").append(this.getZipFile()).append("\n");
+        if (this.getColumns() != null) {
+            sb.append("Columns:");
+            for (DFUDataColumnInfo col : this.getColumns())
+            {
+                sb.append("    ").append(col.getColumnLabel()).append(":").append(col.toString());
```

Comment:
pre-existing, but col.tostring already prints out the columlabel, so we can get rid of the .append(col.getColumnLabel()).append(":")